src/components/Search/beta/SearchDropdown.tsx

```diff
@@ -20,12 +20,46 @@ import {
   DropDownMenuRaw,
   DropDownItemRaw,
 } from '../../../styles';
+import type { Repository } from '../../../types/state';
 
-function SearchDropdown(props: SearchDropdownProps) {
-  const { repository, view, selectLabel, tooltipText, mode } = props;
+function SearchDropdown({
+  repository,
+  view,
+  selectLabel,
+  tooltipText,
+  mode,
+  base,
+}: SearchDropdownProps) {
   const size = view == 'compare-results' ? 'small' : undefined;
   const { handleChangeDropdown } = useHandleChangeDropdown();
 
+  //searchType is to distinguish between base and new search dropdowns for handleChangeDropdown hook
+  const [repoSelect, setRepoSelect] = useState({
+    baseRepository: 'try',
+    newRepository: 'try',
+    searchType: base,
```

Comment:
The `searchType` never changes, so it's not clear to me why you put it in the state. It seems to me this is a prop for the dropdown but will never change for this specific dropdown. Am I missing something?

Also I don't see why we keep both values `baseRepository` and `newRepository`, it looks like to me that we'll ever change one of these values.

Then `handleChangeDropdown` would take 2 parameters `searchType` and `repositoryName`.